server/tests/mochData/socialMockData.js

```diff
@@ -0,0 +1,41 @@
+const mockData = [
+  {
+    username: 'Patsy Waters',
+    firstname: 'Patsy',
+    lastname: 'Waters',
+    email: 'Anais79@hotmail.com',
```

Comment:
I hope that this email does not exist. If so, usea fake email. No need to provide real data in testing and that’s why It’s called, **Mocked Data**. We just follow the desired structure of information for testing purpose.

Reply:
Feedback Implemented! They are using fake emails now.